src/api/tests/core/user/balance.test.ts

```diff
@@ -0,0 +1,66 @@
+import { expect } from 'chai'
+import app from '../../../app'
+import supertest from 'supertest'
+import { constants } from 'sakiewka-crypto'
+// @ts-ignore
+const { user } = app.sakiewkaApi
+
+// @ts-ignore
+const mockFn = jest.fn().mockResolvedValue('balbace')
+
+user.balance = mockFn
+
+describe('/user/balance', () => {
+  beforeEach(() => {
+    mockFn.mockClear()
+  })
+
+  it('should not accept request with missing header', async () => {
+    const response = await supertest(app)
+      .get(`/${constants.BASE_API_PATH}/user/balance`)
+      .send({ fiatCurrency: 'USD' })
+
+    expect(response.status).to.be.equal(400)
+    expect(response.body.errors[0].message).to.be.equal('Request header Authorization is required.')
+  })
+
+  it('should not accept request without authorisation', async () => {
```

Comment:
What's the difference between this and the previous one?

Reply:
No difference - it's duplicate. Thanks, I will remove one of them.